lang/main.json

```diff
@@ -304,6 +304,7 @@
         "connecting": "Connecting",
         "contactSupport": "Contact support",
         "copied": "Copied",
+        "copiedMeeting": "Copied meeting link",
```

Comment:
Hum, this is bad english. But also, why not just use the "Copied" text?

Reply:
that's also fine I'll change that to "copiedMeeting": "Copied", this ?

Reply:
Don't add a new key, just used the existing one please.